File: pgm/ABC_Dart/ABC_342_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';
// import 'dart:typed_data';

void main() {
  final String S = stdin.readLineSync()!;
  const int only = 1;
  const int one = 1;
  const int zero = 0;
  final Map<String, int> map = {};
  for (int i = zero; i < S.length; i++) {
    map[S[i]] = (map[S[i]] ?? zero) + one;
  }
  for (final String k in map.keys) {
    if (map[k] == only) {
      print(S.indexOf(k) + one);
      break;
    }
  }
  exit(0);
}

File: pgm/ABC_Kotlin/ABC_342_A.kt
// import java.time.*
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.util.StringTokenizer

// import kotlin.math.*

const val only: Int = 1
const val one: Int = 1
const val zero: Int = 0

fun main() {
    val S: String = readln()
    val map: MutableMap<Char, Int> = mutableMapOf()
    for (i: Int in zero until S.length) {
        map[S[i]] = map.getOrDefault(S[i], zero) + one
    }
    for (k: Char in map.keys) {
        if (map[k] == only) {
            println(S.indexOf(k) + one)
            break
        }
    }
    kotlin.system.exitProcess(0)
}